tests/forte/forte_backbone_test.py

```diff
@@ -56,3 +56,10 @@ def test_import_models(self):
 
```

Comment:
Some of these import tests are not necessary. For example, if we want to use the `SRL` part we should install texar-pytorch. If we don't use that part, classes like `SRLSpan` won't be used.

Reply:
Even though some classes under `SRL` are not using `texar-pytorch` directly but the whole module dependent on packages required by `forte[models]`. Therefore, we don't need to test any of them as they are not used individually. Is that what you mean?

Reply:
yes

Reply:
ok I have removed them.